src/lib/components/molecules/page-section/stories.module.scss

```diff
@@ -0,0 +1,47 @@
+h2 {
+  font-family: var(--text-headline);
+  font-size: var(--headline-xxsmall);
+  line-height: var(--headline-line-height);
+  font-weight: 700;
```

Comment:
quick dark mode question. Should we add dark mode colors eg `color: var(--primary-text-color);` at a low-level inside individual components, or should that be the responsibility of the host app setting eg h2 colors? 

what do you think?

Reply:
I think components should probably do that, or at least have some sensible defaults. This styling however is not applied to the component. It's only for the Storybook examples.